lib/Sema/DeclSpec.cpp

```diff
@@ -1051,6 +1053,12 @@ bool DeclSpec::setSpecItypeforany(SourceLocation Loc, const char *&PrevSpec, uns
   return false;
 }
 
+bool DeclSpec::setUnpackSpec(SourceLocation Loc, const char *&PrevSpec, unsigned &DiagID) {
```

Comment:
Does this need error checking? If not, please add a comment as to why not.  I am concerned about combining this with other specifiers.